reconciler/testing/reactions_test.go

```diff
@@ -54,17 +56,15 @@ func TestInduceFailure(t *testing.T) {
 		resource:    "inmemorychannels",
 		wantHandled: true,
 	}, {
-		name:        "resource, wrong verb",
-		verb:        "create",
-		testSource:  imc,
-		resource:    "inmemorychannels",
-		wantHandled: false,
+		name:       "resource, wrong verb",
```

Comment:
it would be better as a `map[string]struct` for the test name.

Reply:
No, I strongly disagree :)